src/components/common/deleteResource.jsx

```diff
@@ -97,14 +97,16 @@ DeleteResourceModal.propTypes = {
     onClose: PropTypes.func.isRequired,
 };
 
-export const DeleteResourceButton = ({ objectId, disabled, overlayText, actionName, showDialog }) => {
+export const DeleteResourceButton = ({ objectId, disabled, overlayText, actionName, showDialog, isLink, isInline }) => {
     if (disabled) {
         return (
             <Tooltip id={`delete-${objectId}-tooltip`}
                      content={overlayText}>
                 <span>
                     <Button id={`delete-${objectId}`}
-                        variant='danger'
+                        variant={isLink ? 'link' : 'danger'}
+                        isDanger={isLink}
```

Comment:
This still looks rather confusing .. but indeed that's just the [API](https://www.patternfly.org/v4/components/button/).